docs/src/token-swap.md

```diff
@@ -53,28 +53,28 @@ The following explains the instructions available in the Token Swap Program.
 Note that each instruction has a simple code example that can be found in the
 [end-to-end tests](https://github.com/solana-labs/solana-program-library/blob/master/token-swap/js/cli/token-swap-test.js).
 
-### Creating a new token swap
+### Creating a new token swap pool
```

Comment:
I think it will be easier to find if host fee address in this file in addition to README.md since this file is published to  spl.solana.com. Additionally, it might be worth adding a table at the top with token-swap address itself for easy of discovery.